src/main/resources/ownership.json

```diff
@@ -279,6 +355,85 @@
         "path": "Azure/*",
         "branch": "*"
       }
+    ]
+  },
+  {
+    "name": "Moderne",
+    "repositories": [
+      {
+        "origin": "github.com",
+        "path": "moderneinc/*",
+        "branch": "*"
+      }
+    ],
+    "commitOptions": [
+      "PullRequest",
+      "Branch"
+    ]
```

Comment:
The problem here is, that although Moderne staff can pr directly, this isn't possible for others external to Moderne.

Reply:
You won't be able to do any of these actions since this is for Moderne private code.

Reply:
So this was intentional